test/performance/search/search_benchmark.cpp

```diff
@@ -162,8 +162,13 @@ void unidirectional_search_all_collection(benchmark::State & state, options && o
     seqan3::fm_index index{collection};
     seqan3::configuration cfg = seqan3::search_cfg::max_error_total{seqan3::search_cfg::error_count{o.searched_errors}};
 
+    size_t sum{};
     for (auto _ : state)
+    {
         auto results = search(reads, index, cfg);
+        sum += std::ranges::distance(results);
```

Comment:
```suggestion
        sum += std::ranges::distance(results); // Forces the actual computation of the search through the lazy algorithm result range.
```

Reply:
Ok, this is at every place. Could you just put that in the top of the benchmark file. So one can read up on why we do this distance computation? Then this comment must not be repeated for every benchmark